### ISSUE TITLE
Request: Color coding in preview file

### ISSUE THREAD

**Issue by Nachtinho** [Workiva/ixbrl-viewer#326]
Hi,

Currently, all tags are marked as “green” even if tags overlap. This makes it less readable and distinguishable.

It would be nice to have more different colors (Green, Blue, Purple, Grey), depending on how many tags are overlapping.

Kind regards,

**Comment by paulwarren-wk** [comment]
Yes, this is something that I'd like to address.  We currently use different colours for different namespaces, which is useful for quickly identifying extensions.

I've been considering using different shades to distinguish nested facts – we don't need colors to indicate meaning here, just something to differentiate so that it's possible to see that there's an additional tag there.  I was hoping that this could be done easily by specifying partially transparent colors, but results of my initial experimenting were far too subtle.
